### DISCUSSION TITLE
question about two interactivity features

### DISCUSSION THREAD

**Discussion by gdanov** [observablehq/plot#456] [Ideas]
Hi,
 I looked through the intro and the samples but could not see two use cases illustrated:

- scrolling
- annotating charts

 Is there built-in support for that? If not — is it planned, is it supposed to be done via plug-in, etc.?

Thanks,
Georgi

**Comment by Fil** [comment]
For annotations it's already possible to add text marks arbitrarily, for a simple example see https://observablehq.com/@fil/plot-stargazer-history — however I suppose that you might have something more specific in mind, maybe like @susielu's https://d3-annotation.susielu.com/ ?

Also, what examples of scrolling interactivity would you recommend?
